src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/runMatlabTests.m

```diff
@@ -1,6 +1,6 @@
-%Copyright 2019-2020 The MathWorks, Inc.
+%Copyright 2020 The MathWorks, Inc.
 
-function failed = runMatlabTests(varargin)
+function runMatlabTests(varargin)
```

Comment:
Is this function necessary? The reason I ask, is it's going to add to the users stack trace. 

Could you just run the contents as the command itself? That's not totally ideal either though because then we can get into parsing trouble. I think maybe we should discuss.

Reply:
To add to it this file will anyways get deleted in either case. May be stack trace, in that case, would be helpful. will discuss this i will keep it open till we conclude.

Reply:
Nikhil and I discussed this offline. 

The pro of writing a file to disk is that we do not have to worry about parsing issues with injecting user provided text into a shell command input. The con is that the file becomes part of the users stack trace.

Neither felt like a clear win, so we will keep this as is for now.

Reply:
Yes and we shall continue persuading this till we conclude on a robust solution  I have raise #133 to track the same